install/kubernetes/cilium/values.yaml.tmpl

```
@@ -3340,6 +3340,10 @@ cgroup:
     #     memory: 128Mi
   # -- Configure cgroup root where cgroup2 filesystem is mounted on the host (see also: `cgroup.autoMount`)
   hostRoot: /run/cilium/cgroupv2
+# -- Configure sysctl override described in #20072.
+sysctlfix:
+  # -- Enable the sysctl override. When enabled, it temporally mounts the /proc of the host from init container.
```

```suggestion
  # -- Enable the sysctl override. When enabled, the init container will mount the /proc of the host so that the `sysctlfix` utility can execute.
```

Nit: It took me a couple of reads to parse this, maybe this is a little clearer?